CONTRIBUTING.md

```diff
@@ -2,3 +2,10 @@
 
 See the [contributing section of the
 website](https://metallb.universe.tf/community) for information.
+
+## Certificate of Origin
+
+By contributing to this project you agree to the Developer Certificate of
+Origin (DCO). This document was created by the Linux Kernel community and is a
+simple statement that you, as a contributor, have the legal right to make the
+contribution. See the [DCO](DCO) file for details.
```

Comment:
Just curious: this link works because it is in the same directory? Don't we need a `./` nor anything?

Reply:
yeah, same directory.  It works on github, at least! Check out: https://github.com/russellb/metallb/blob/dco/CONTRIBUTING.md

Reply:
Thanks!